src/v1/destinations/monday/networkHandler.js

```diff
@@ -0,0 +1,108 @@
+const { TransformerProxyError } = require('../../../v0/util/errorTypes');
+const { proxyRequest, prepareProxyRequest } = require('../../../adapters/network');
+const {
+  processAxiosResponse,
+  getDynamicErrorType,
+} = require('../../../adapters/utils/networkUtils');
+const { isHttpStatusSuccess } = require('../../../v0/util/index');
+const tags = require('../../../v0/util/tags');
+
+const checkIfUpdationOfStatusRequired = (response) => {
+  let errorMsg = '';
+  const responseBodyStatusCode = response.status_code;
+  if (
+    response.hasOwnProperty('error_message') ||
+    response.hasOwnProperty('error_code') ||
+    response.hasOwnProperty('errors')
+  ) {
+    errorMsg = response.error_message || response.errors?.map((error) => error.message).join(', ');
```

Comment:
errors going to always array only? use response.errors?.map?.((error) => error.message).join(', ');

Reply:
yes, as per documentation, it's always array